runtime/lua/vim/lsp/handlers.lua

```diff
@@ -217,6 +217,11 @@ RCS[ms.textDocument_diagnostic] = function(...)
   return vim.lsp.diagnostic.on_diagnostic(...)
 end
 
+--- @private
+RCS[ms.textDocument_foldingRange] = function(...)
+  return vim.lsp.folding_range.on_folding_range(...)
+end
```

Comment:
We don't add handlers here anymore. Instead this should be inlined into the fold module.